libs/sdk/e2e-schematics/src/generators/configure-percy/__snapshots__/index.spec.ts.snap

```diff
@@ -1,14 +1,14 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`configure-percy should generate cypress.config.ts 1`] = `
-"import { defineConfig } from 'cypress';
-import { nxE2EPreset } from '@nrwl/cypress/plugins/cypress-preset';
+"import { nxE2EPreset } from '@nx/cypress/plugins/cypress-preset';
```

Comment:
There were some changes like in this file where NX/prettier just formats differently than they used to - no functional changes.